modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -161,7 +161,7 @@
   nil)
 
 (defn- throw-invalid-query [e sql parameters]
-  (throw (ex-info (tru "Error executing query")
+  (throw (ex-info (tru "Error executing query: {0}" (ex-message e))
```

Comment:
Unrelated to this fix but I changed a few errors to include the parent error message to make query failures easier to debug. The updated error message is already used elsewhere so it's already i18n'ed